hudi-client/src/main/java/org/apache/hudi/client/HoodieReadClient.java

```diff
@@ -100,14 +91,6 @@ public HoodieReadClient(JavaSparkContext jsc, String basePath, SQLContext sqlCon
    * @param clientConfig instance of HoodieWriteConfig
    */
   public HoodieReadClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig) {
-    this(jsc, clientConfig, Option.empty());
-  }
-
-  /**
-   * @param clientConfig instance of HoodieWriteConfig
-   */
-  public HoodieReadClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig,
-      Option<EmbeddedTimelineService> timelineService) {
```

Comment:
so this was not doing anything after we changed the class hierrarchy? and this PR is cleaning this up?

Reply:
Yes

Reply:
Okay sg. its simpler for now..  cc @bvaradar as fyi.. you may run into this if interested in de-duping..

Reply:
Thanks @vinothchandar  for pointing me to this change. @yanghua : Just catching up on this. Instead of completely removing EmbeddedTimelineService from HoodieReadClient, did you try removing SparkConf dependency from  EmbeddedTimelineService thereby also making it generic. I wanted to make sure all APIs of hudi client go through Embedded timeline service so that we can guarantee meaningful advantage when using timeline service in future.

Reply:
@bvaradar I and @vinothchandar discussed before, we agreed on that `HoodieReadClient ` will be moved into hudi-spark module in the future. Currently, we found {{EmbeddedTimelineService}} did not take effect in `HoodieReadClient`.

Reply:
@yanghua : I am ok as long as with HUDI-661, we are bringing back TimelineService support for ReadClient. We need to make sure HUDI-661 is also addressed quickly before next release (0.6.0).  I understand that refactoring these pieces is sometimes tricky and am ok with your approach.

Reply:
@bvaradar Will try to return back soon, I am helping to release v0.5.2.